autoload/operator/surround.vim

```diff
@@ -233,6 +233,9 @@ function! s:surround_blocks(block_begin, block_end)
     let [_, last_line, end_col, _] = getpos("']")
     let is_extended = s:is_extended_blockwise_visual()
     for line in range(start_line, last_line)
+        if getline(line) ==# ''
+          continue
+        endif
```

Comment:
How about a line containing only white spaces?

```vim
if line =~# '^\s*$'
```

may be more useful. What do you think?

Reply:
I agree, it's useful to ignore lines consisted of only whitespace characters also. I updated pr.